sdk/resourcemanager/azure-resourcemanager-test/src/main/java/com/azure/resourcemanager/test/ResourceManagerTestProxyTestBase.java

```diff
@@ -467,8 +467,14 @@ private void addSanitizers() {
             // ContainerRegistry
             new TestProxySanitizer("$..passwords[*].value", null, REDACTED_VALUE, TestProxySanitizerType.BODY_KEY),
             // ContainerService
-            new TestProxySanitizer("$..secret", null, REDACTED_VALUE, TestProxySanitizerType.BODY_KEY)
-        ));
+            new TestProxySanitizer("$..secret", null, REDACTED_VALUE, TestProxySanitizerType.BODY_KEY),
+            // AppService
+            new TestProxySanitizer("$.properties.siteConfig.machineKey.decryptionKey", null, REDACTED_VALUE, TestProxySanitizerType.BODY_KEY),
+            // Replace "AccountKey=<accountKey>;" with "AccountKey=REDACTED;"
+            new TestProxySanitizer("(?:AccountKey=)(?<accountKey>.*?)(?:;)", REDACTED_VALUE, TestProxySanitizerType.BODY_REGEX).setGroupForReplace("accountKey"),
+            new TestProxySanitizer("$.properties.WEBSITE_AUTH_ENCRYPTION_KEY", null, REDACTED_VALUE, TestProxySanitizerType.BODY_KEY),
```

Comment:
Got two more confirmed. 
Currently there's some issue with FunctionAppTest for ACA, reported to service team.
Already confirmed with Vinay that the related resources has been deleted. Guess it's fine to update recordings afterwards.

Reply:
Manual updated session records. Also deleted two test records whose test is marked `@Disabled`.